ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -26,7 +26,6 @@ public partial class PrincipiaPluginAdapter
       "principia_initial_state";
   private const String principia_gravity_model_config_name =
       "principia_gravity_model";
-  private const double Δt = 10;
```

Comment:
In light of the "Δt trainwreck" I am going to ask that all the member variables at lines 19-28 be renamed to have a trailing underscore.

Reply:
Done.